CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to
 - An `errorCode`, `errorId`, and `reason` is now displayed in step failure
   events and logs.
 - Event logging around errors thrown in `validateInvocation` function.
+- Abort synchronization job when `run` and `sync` command fails.
```

Comment:
I think this is a good example of why we should consider separating the execution code into a separate package from the code the integrations themselves depend on. Should we have a bug in the runner, we can avoid upgrading all the open source integrations or worse, not upgrading them and leaving them on old versions.

Reply:
cc/ @philidem